src/main.rs

```diff
@@ -105,6 +105,9 @@ enum Command {
         /// Delete without prompting.
         #[arg(short, long)]
         force: bool,
+        /// Delete a row in the usual records table or the null records table.
+        #[arg(short, long)]
+        null: bool,
```

Comment:
Is this meant to say "Delete a row in the null records table"?

But actually, I think it might be useful to handle multiple citation keys here (in a future PR), so I would instead do a `--delete-null` option, with enables deleting null records as well as usual records.